### PR TITLE
Move adjustments to CSV files to extensions.

### PR DESCRIPTION
**Pull request by elupus**

### CODE REVIEW

**Comment by elupus** [comment]
I'm not fully happy with this. Maybe all keys should be sorted so we can keep order easier. Now extension are always added to the end of fields.

**Comment by cybermaus** [comment]
Regarding these extentions, for some of the SMOS40 (maybe all S) registers, as I am reading tcp/modbus directly at the moment so I know some of the mappings.
Without expectations, nor pushing them to be included, I leave them here in case you may find them useful.


```
Priority	MODBUS_INPUT_REGISTER	1028	1		4	0	0	0
# 10: Off, 20: Hot Water, 30: Heat, 40: Pool, 50: Cooling
Compressor status (EB101)	MODBUS_INPUT_REGISTER	1484	1		4	0	0	0
# 0: Off, 1: On
External heating medium pump (GP10) status	MODBUS_INPUT_REGISTER	1066	1		4	0	0	0
# 0: Off, 1: On
```

Also, but this is somewhat more complex and probably out of scope for this extension system, if you have a RMU room sensor in the same climate system next to the standard BT50, then you cannot read the 2nd  room sensor (Roomsensor 1-2) directly, but you can calculate it from the average (this is not a time average, but a multiple sensor average). 
```
Roomsensor 1-1	MODBUS_INPUT_REGISTER	26	10	°C	2	0	0	0
Room average temp. clim. system 1 (BT50)	MODBUS_INPUT_REGISTER	116	10	°C	2	0	0	0
#Roomsensor 1-2 = 2*(116)-(26) (or rather, AVG=(BT50+RMU)/2
```

**Comment by elupus** [comment]
I do think we should be able to grab alot of S series mappings from F series. I think they internally are very much the same. I do wonder why they had to change the parameter numbers though :(

**Comment by elupus** [comment]
I was thinking of just ordering everything in the json.dump() call. But need to be separate since its a huge change.

File: nibe/data/f1355.json [hunk 1]
```diff
@@ -8754,10 +8750,7 @@
     "name": "operational-mode-brine-medium-pump-47139",
     "write": true,
     "mappings": {
-      "10": "Intermittent",
-      "20": "Continuous",
-      "30": "Economy",
-      "40": "Auto"
+      "20": "Continuous"
```
Comment (yozik04) [inline nibe/data/f1355.json:8753]: some mapping detection got broken

**Pull request merged**